.ci_support/win_c_compilervs2015cxx_compilervs2015python3.5.yaml

```diff
@@ -9,6 +9,6 @@ pin_run_as_build:
 python:
 - '3.5'
 zip_keys:
-- - c_compiler
+- - python
+  - c_compiler
```

Comment:
did you have to edit these manually ??

Reply:
No, I didn't. I just edited the ```meta.yaml``` the other files were the rerender process